### PR TITLE
Figure 5 edits

### PR DESCRIPTION
**Pull request by Farnazmdi**
In this PR:
1. figure 5 instead of parameters plotted the distance of distributions.
2. figure 5 for `pi` changed the x-axis to the number of lineages.

### CODE REVIEW

**Comment by Farnazmdi** [comment]
image

**Comment by Farnazmdi** [comment]
Here is figure 5 now.
image

**Comment by Farnazmdi** [comment]
@aarmey Should supplementary figures change based on this or they should stay the same?

**Comment by aarmey** [comment]
I think I like keeping the old way in the supplement.

File: lineage/tHMM.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def log_score(self, X_state_tree_sequence, pi=None, T=None, E=None):
             log_score += log_T_score(T, X_state_tree_sequence[idx], lineageObj)
             log_score += log_E_score(get_Emission_Likelihoods(self, E)
                                      [idx], X_state_tree_sequence[idx])
-            assert not np.isnan(log_score), f"log score is nan"
+            np.all(np.isfinite(log_score))
```
Comment (aarmey) [inline lineage/tHMM.py:158]: Want an `assert` in front of this.

**Review by aarmey — APPROVED** [review]
One edit then this looks ready to me.

**Pull request merged**